### PR TITLE
feat(lua): add missing no-op redis.replicate_commands()

### LINKED ISSUE #2468: introduce redis.replicate_commands() in lua
Hi, 

I'm a heavy user of https://github.com/mvisonneau/gitlab-ci-pipelines-exporter and I have been looking to migrate it's redis dependency to dragonflydb.

On Dragonfly 1.10, I was blocked due to missing XACK. Now, i'm trying with version 1.13 and I'm one step further. Now it gives me:

{"error":"ERR Error running script (call to f52a5b566060d79fd321d695df55bc918339c4aa): @user_script:4: user_script:4: attempt to call a nil value (field 'replicate_commands')","level":"fatal","msg":"","time":"2024-01-24T18:16:39Z"}

I googled quickly and it looks like a redis command. I will dive more but if someone knows about it and want to save my time it would be great. 

thanks! btw, great project.

### PR DESCRIPTION
**Pull request by kostasrim**
fixes #2468

* add no-op lua function redis.replicate_commands()

### CODE REVIEW

**Comment by EvertonSA** [comment]
amazing brotherrr

**Comment by EvertonSA** [comment]
if there is a docker i can test, let me know

**Comment by kostasrim** [comment]
> if there is a docker i can test, let me know

we got a weekly docker build. Not sure if @romange wants to trigger this earlier

File: tests/dragonfly/eval_test.py [hunk 1]
```diff
@@ -264,3 +264,12 @@ async def test_lua_auto_async(async_client: aioredis.Redis):
 
     flushes = (await async_client.info("transaction"))["eval_squashed_flushes"]
     assert 1 <= flushes <= 3  # all 100 commands are executed in at most 3 batches
+
+
+@dfly_args({"proactor_threads": 4, "lua_auto_async": None})
```
Comment (romange) [inline tests/dragonfly/eval_test.py:269]: I prefer testing this via `interpreter_test.cc` no need for regression tests for this :)

**Review by adiholden — APPROVED** [review]

**Pull request merged**